Rebus.AdoNet/Dialects/PostgreSqlDialect.cs

```diff
@@ -46,6 +46,7 @@ public PostgreSqlDialect()
 
 		#region Overrides
 		public override bool SupportsSelectForUpdate => true;
+		public override bool SupportsPgTryAdvisoryLock => true;
 		public override string ParameterSelectForUpdate => "FOR UPDATE";

```

Comment:
You probably want a ParameterSkipLocked similiar to this ParameterSelectForUpdate, along with a SupportsSkipLocked property.